sigal/writer.py

```diff
@@ -81,6 +83,9 @@ def __init__(self, settings, index_title=''):
             **env_options
         )
 
+        for filter in filters:
+            env.filters[filter.__name__] = filter
```

Comment:
It could indeed be linked to the templates, but as there are not many themes/templates I think it's easier for now to put everything in `sigal.conf.py`.